### ISSUE TITLE
Saving issue

### ISSUE THREAD

**Issue by nad45** [SonicMastr/rePatch-reLoaded#2]
Hello
I tried repatch with Max Payne, I moved the data folder in a new location. 
I can play the game but when I want to save, it doesn't work. I think it still wants to save in ux0/data/Maxpayne not in the new data folder's location. 
Same with Gtasa
Is it a solvable problem? 
Thanks

**Comment by ShroomKing** [comment]
Yeah, updating files works fine but when programs create a new file they try to do it in ux0:/data/.../ but it fails because the folder doesn't exist anymore.

Same thing happened to me with reVC and the txd.img and txt.dir because i moved the data directory to uma0:
To work around this i made an empty directory (ux0:/data/gtavc/models) to let it create the files and then moved them over to uma0: later. 
The same workaround should work for saves too.

Obviously the ideal way would be to redirect created files to reData, but no idea if it's possible.

**Comment by KoleckOLP** [comment]
Yeah I have the same issue with Vice City and Fallout 2 CE
in Fallout 2 CE I am unable to load or save the game, but starting a new game works.

I'd say it's pretty much unusable for homebrew right now.
